File: Java/src/Perfect_Number.java
/**
 * Perfect Number Checker
 * 
 * @author Jaysmito Mukherjee 
 * @version 2019-A-009
 */
import java.util.*;
public class Perfect_Number
{
    public static void main()
    {
        System.out.print("\f");
        Scanner sc=new Scanner(System.in);
        System.out.print("Perfect Number Checker\n~~~~~~~ ~~~~~~ ~~~~~~~\nEnter the number: ");
        int n=sc.nextInt();
        int s=0;
        for(int i=1;i<=n/2;i++)
            if(n%i==0)
            s=s+i;
        if(s==n)
            System.out.print(n+" is a Perfect number.");
        else
            System.out.print(n+" is not a Perfect number.");        
    }
}

File: Python/Perfect_Number.py
print("Perfect Number Checker\n")
n = int(input("Enter the number : "))
s = 0
for i in range(1, n//2+1) :
	if n%i == 0:
		s += i
if s == n:
	print(f"{n} is a perfect number.")
else:
	print(f"{n} is not a perfect number.")